### ISSUE TITLE
Task page redirection from the model search page

### ISSUE THREAD

**Issue by merveenoyan** [huggingface/huggingface_hub#614]
Hello,
I think it's a UX improvement and an improvement for better visibility of the pages. I want a redirection from the models page to task page, such that when they people filter for a task, they can also get better information on it (+these resources would be more visible)

<img width="1028" alt="Ekran Resmi 2022-01-20 19 32 22" src="https://user-images.githubusercontent.com/53175384/150381086-a1d4f41f-f5f3-4055-94de-e01992456244.png">

**Comment by beurkinger** [comment]
What we could do is having some sort of link/button next to the "Tasks" filter category that redirect to the /tasks page ?

**Comment by beurkinger** [comment]
Idk how relevant it is on this page

**Comment by merveenoyan** [comment]
> What we could do is having some sort of link/button next to the "Tasks" filter category that redirect to the /tasks page ?

yes that's what I thought
also @julien-c @jeffboudier since we talked about it today (maybe they might have an additional suggestion)

**Comment by jeffboudier** [comment]
Personally I wouldn't have it on the default /models sidebar, it doesn't feel contextual enough, my suggestion would be to have the link after the user clicks on the list overflow button [+13], then you're really considering all the different tasks and interested about what else HF offers in tasks

And maybe so we don't add cognitive load instead of adding a linked label (e.g. "[learn more](https://huggingface.co/tasks)"), to link the "Tasks" sidebar header label itself (underline on hover for affordance, as in the task name in the inference widget)

**Comment by osanseviero** [comment]
I agree with you @jeffboudier that having in the /models sidebar by default does not feel right. 

One of the concerns with the underline on hover is that it's not discoverable at all. Maybe not a linked text, but for me a ? icon next to "Tasks" **only** after clicking [+13] would be a good trade-off between discoverability and cognitive overload.

**Comment by merveenoyan** [comment]
Agreed. That's what we currently have in the widgets and I think it's not good for visibility.

**Comment by jeffboudier** [comment]
Yes I understand @merveenoyan @osanseviero - and what's good for visibility of one feature is bad for the overall cognitive load. The (?) icon makes sense functionally (I think more so on the /models expanded task sidebar vs. inference widget task link, because it's more directly connected to what user explicitly wants to do), but I haven't seen us use that UI much across the site (most labels on the hub could also have a (?) next to them), underline on-hover seems more consistent to me but in the end that's really a design question @gary149

**Comment by adrinjalali** [comment]
Any conclusions here? (cleaning up the repo ref: https://github.com/huggingface/huggingface_hub/issues/744)
